app/code/Magento/SendFriend/Helper/Data.php

```diff
@@ -49,6 +53,36 @@ public function isEnabled($store = null)
         );
     }
 
+    /**
+     * Check if email should be sent from store
+     *
+     * @param int $store
+     * @return bool
+     */
+    public function isSentFromStore($store = null)
```

Comment:
Unfortunately, we are not able to add new public methods to the @api classes according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)